### PR TITLE
Update release notes 14.1

### PR DESCRIPTION
**Pull request by corinnaguenther**
[#54231] Release notes prepared

https://community.openproject.org/work_packages/54231

### CODE REVIEW

File: docs/release-notes/14-1-0/README.md [hunk 1]
```diff
@@ -16,16 +16,20 @@ We released [OpenProject 14.1.0](https://community.openproject.org/versions/2030
...
+This feature was requested by many in the Community and will be a great benefit, especially for organizations with many projects: OpenProject 14.1 allows you to add projects to favorites so that they can be found more quickly. We [recently gave an overview in our blog](https://www.openproject.org/blog/favorite-projects/) of how exactly you can add projects to favorites or remove them from favorites – and where you can display them.
 
 ![OpenProject dropdown menu showing favorite projects, displayed with yellow stars](openproject-14-1-favorite-projects-overview.shadow.png)
 
+See our documentation to learn more about [how to mark a project as favorite](https://www.openproject.org/docs/user-guide/project-overview/#mark-a-project-as-favorite).
+
 ### Advanced features for the Meetings module
 
 The Meetings module is currently being continuously improved. With OpenProject 14.1...
```
Comment (MayaBerd) [inline docs/release-notes/14-1-0/README.md:35]: Did you want to mention previous changes to the Meetings Module here?

File: docs/release-notes/14-1-0/README.md [hunk 2]
```diff
@@ -39,15 +43,19 @@ The Meetings module is currently being continuously improved. With OpenProject 1
 Apart from these features that apply the Meetings module directly, the following changes concerning OpenProject meetings have been made:
 
 - On work packages, the project name is now shown in the Meetings tab.
```
Comment (MayaBerd) [inline docs/release-notes/14-1-0/README.md]:
```suggestion
- In work packages, the project name is now shown in the Meetings tab.
```

File: docs/release-notes/14-1-0/README.md [hunk 3]
```diff
@@ -39,15 +43,19 @@ The Meetings module is currently being continuously improved. With OpenProject 1
...
-- Widgets to show meetings can be added to the My Page and project overview pages.
+- Calendar widgets on the My Page and project overview pages can now show meetings.
+
+![A calendar widget on a project overview page, now showing upcoming meetings](openproject-14-1-meetings-widget-project-overview.png)
+
+**Please note**: On the project overview page, the "+" button on the top right has been removed. You can now only add widgets by hovering at the border of an existing widget and click on the "+" there. 
 
 ### Possibility to hide attachments in the Files tab
 
 Admins now are able to hide the attachment section in the Files tab. This setting can be made at project level and is particularly useful for projects where users should only upload files via external storage, e.g. Nextcloud.
```
Comment (MayaBerd) [inline docs/release-notes/14-1-0/README.md]:
```suggestion
Admins now are able to hide the attachment section in the Files tab. This setting can be changed at a project level and is particularly useful for projects where users should only upload files via external storage, e.g. Nextcloud.
```

**Comment by MayaBerd** [comment]
Some links to documentation are missing (because they are not merged yet). I am happy to add them once documentation is finalized.

File: docs/release-notes/14-1-0/README.md (hunk 1 above)
Comment (corinnaguenther) [reply docs/release-notes/14-1-0/README.md:35]: Not in detail, I just wanted to show that it is a focus topic for us right now

**Pull request merged**